### PR TITLE
Added the tensorflow pip package

### PR DESCRIPTION
**Pull request by peci1**
https://pypi.python.org/pypi/tensorflow/0.12.1.

Google's framework for neural networks. This only provides the python interface; the C++ API is trickier to get.

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -2560,6 +2560,10 @@ python-telegram-bot:
   ubuntu:
     pip:
       packages: [python-telegram-bot]
+python-tensorflow:
```
Comment (tfoote) [inline rosdep/python.yaml]: Please use `-pip` extension for pip only packages. Pip is often well ahead of the system packages when they eventually appear in the future which can cause future breakage.

**Review by tfoote — CHANGES_REQUESTED** [review]
Since it's pip only, it might be helpful for others to add rules for other common platforms such as fedora, debian and osx.

**Comment by peci1** [comment]
Updated.

File: rosdep/python.yaml [hunk 2]
```diff
@@ -2560,6 +2560,19 @@ python-telegram-bot:
   ubuntu:
     pip:
       packages: [python-telegram-bot]
+tensorflow-pip:
```
Comment (dirk-thomas) [inline rosdep/python.yaml]: Should the key be `python-tensorflow-pip`?

Comment (peci1) [inline rosdep/python.yaml]: Ahh, sorry :)

**Review by dirk-thomas — APPROVED** [review]

**Review by tfoote — APPROVED** [review]

**Pull request merged**